pyrsa/rdm/rdms.py

```diff
@@ -463,9 +464,58 @@ def rank_transform(rdms, method='average'):
                                 for i in range(rdms.n_rdm)])
     rdms_new = RDMs(dissimilarities,
                     dissimilarity_measure=rdms.dissimilarity_measure,
-                    descriptors=rdms.descriptors,
-                    rdm_descriptors=rdms.rdm_descriptors,
-                    pattern_descriptors=rdms.pattern_descriptors)
+                    descriptors=deepcopy(rdms.descriptors),
+                    rdm_descriptors=deepcopy(rdms.rdm_descriptors),
+                    pattern_descriptors=deepcopy(rdms.pattern_descriptors))
+    return rdms_new
+
+
+def sqrt_transform(rdms):
```

Comment:
Also wouldn't this be more attractive as a method?

Reply:
we could use it as a method, too. Would it then make sense to interpret this as an in place operation? If so: Should we then use this as the convention?
`function(object)` returns a new object
`object.function()` changes the object

Didn't want to start such a convention without checking in.

Reply:
Yeah that's a good question. I was thinking to open an issue about this a while ago. I'm fine either way as long as we aim to keep it consistent. Also if possible the methods would return `self` so you can chain them: `rdms.sum().sqrt()`.

Reply:
I've noticed this applied to a few other transformations of RDMs and data. I will copy this into an issue about this and fix this at all places simultaneously and merge this pull request for now.